python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -353,3 +353,95 @@ def _doConvolve(exposure, kernel):
         afwMath.convolve(outExp.getMaskedImage(), exposure.getMaskedImage(), kern, convCntrl)
 
         return outExp, kern
+
+
+class DecorrelateALKernelMapperTask(DecorrelateALKernelTask):
+    """Task to be used as an ImageMapperSubtask for computing
+    A&L decorrelation on subimages on a grid across a A&L difference image.
+
+    This task subclasses DecorrelateALKernelTask in order to implement
+    all of that task's configuration parameters, as well as its `run` method.
+    """
+    ConfigClass = DecorrelateALKernelConfig
```

Comment:
I encourage you to undo that change. I am not aware of any code in the stack that uses "Subtask" in the name and it seems unnecessary. Any task can be treated as a subtask. Only subclasses of `CmdLineTask` can be top-level tasks, but all other tasks are still considered tasks.